### ISSUE TITLE
hugo convert toTOML fails

### ISSUE THREAD

**Issue by drguildo** [gohugoio/hugo#3844]
```
~/Data/Web Sites/sjm.io$ hugo version
Hugo Static Site Generator v0.26 windows/amd64 BuildDate: 2017-08-28T16:18:54+01:00
~/Data/Web Sites/sjm.io$ hugo convert toTOML -o tomltmp
processing 4 content files
2017/08/28 16:19:27 mkdir C:\Users\Simon\Data\Web Sites\sjm.io\tomltmp\C:: The filename, directory name, or volume label syntax is incorrect.
panic: mkdir C:\Users\Simon\Data\Web Sites\sjm.io\tomltmp\C:: The filename, directory name, or volume label syntax is incorrect.


goroutine 1 [running]:
log.Panicln(0xc042235718, 0x1, 0x1)
        /usr/local/go/src/log/log.go:336 +0xc7
github.com/gohugoio/hugo/vendor/github.com/spf13/afero.WriteReader(0xf7ab00, 0xfe95c0, 0xc0420c05b0, 0x62, 0xf6e040, 0xc0423f2990, 0xf6f880, 0xc0423f2ba0)
        /Users/bep/go/src/github.com/gohugoio/hugo/vendor/github.com/spf13/afero/util.go:49 +0x262
github.com/gohugoio/hugo/helpers.WriteToDisk(0xc0420c05b0, 0x62, 0xf6e040, 0xc0423f2990, 0xf7ab00, 0xfe95c0, 0x3d, 0x14f)
        /Users/bep/go/src/github.com/gohugoio/hugo/helpers/path.go:577 +0x64
github.com/gohugoio/hugo/hugolib.(*Page).saveSource(0xc042078000, 0xc0423e3200, 0x10c, 0x10c, 0xc0420c05b0, 0x62, 0x0, 0xc042235898, 0x536784)
        /Users/bep/go/src/github.com/gohugoio/hugo/hugolib/page.go:1402 +0x35f
github.com/gohugoio/hugo/hugolib.(*Page).saveSourceAs(0xc042078000, 0xc0423db2c0, 0x3d, 0x0, 0x0, 0x0)
        /Users/bep/go/src/github.com/gohugoio/hugo/hugolib/page.go:1391 +0x1cb
github.com/gohugoio/hugo/hugolib.(*Page).SaveSourceAs(0xc042078000, 0xc0423db2c0, 0x3d, 0xc0423db2c0, 0x3d)
        /Users/bep/go/src/github.com/gohugoio/hugo/hugolib/page.go:1378 +0x4b
github.com/gohugoio/hugo/commands.convertContents(0xc00000002b, 0xbec802, 0x1)
        /Users/bep/go/src/github.com/gohugoio/hugo/commands/convert.go:144 +0xba9
github.com/gohugoio/hugo/commands.glob..func2(0xfc2560, 0xc042003160, 0x0, 0x2, 0x0, 0x0)
        /Users/bep/go/src/github.com/gohugoio/hugo/commands/convert.go:56 +0x61 github.com/gohugoio/hugo/vendor/github.com/spf13/cobra.(*Command).execute(0xfc2560, 0xc0420030e0, 0x2, 0x2, 0xfc2560, 0xc0420030e0)
        /Users/bep/go/src/github.com/gohugoio/hugo/vendor/github.com/spf13/cobra/command.go:649 +0x45e
github.com/gohugoio/hugo/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xfc3520, 0xc2cc60, 0x4043d3, 0xc04201c178)
        /Users/bep/go/src/github.com/gohugoio/hugo/vendor/github.com/spf13/cobra/command.go:728 +0x340
github.com/gohugoio/hugo/commands.Execute()
        /Users/bep/go/src/github.com/gohugoio/hugo/commands/hugo.go:173 +0x67
main.main()
        /Users/bep/go/src/github.com/gohugoio/hugo/main.go:27 +0x3d
```

**Comment by drguildo** [comment]
Problem still exists in `Hugo Static Site Generator v0.31.1 windows/amd64 BuildDate: 2017-12-06T10:00:50Z`.

**Comment by drguildo** [comment]
Seems to be fixed in 0.37.
